strawberry/subscriptions/protocols/graphql_transport_ws/handlers.py

```diff
@@ -228,15 +228,12 @@ async def operation_task(
         Operation task top level method.  Cleans up and de-registers the operation
         once it is done.
         """
+        task = asyncio.current_task()
+        assert task is not None
```

Comment:
is this just for mypy?

Reply:
Yes, exactly.  
Btw, in a future PR, I aim to get rid of this code, since it isn't necessary to "reap" background tasks.  This is something I realized recently, I was confused with how "coroutines" must always be awaited.  But this PR focuses on getting rid of current warnings from the unit tests.